lib/plugins.js

```diff
@@ -274,6 +282,8 @@ function make_delegate (instance, plugin) {
     actmeta.plugin_name = plugin.name || '-'
     actmeta.plugin_tag = plugin.tag || '-'
     actmeta.plugin_fullname = plugin.fullname
+
+    // TODO: is this necessary?
```

Comment:
Can you add an issue for this? I want to track it, see can we get a better test around it.